### PR TITLE
integrate geometry, surface and volume mesher

### PR DESCRIPTION
**Pull request by feilin-flexcompute**
- [x] add version for surface mesher
- [x] add/modify key-value in meshers for DB
- [x] add examples for integration tests from geometry to simulation

### CODE REVIEW

File: examples/simulation_examples/airplane_case.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import flow360 as fl
+from flow360 import units as u
+
+
+def createBaseParams_airplane():
```
Comment (maciej-flexcompute) [inline examples/simulation_examples/airplane_case.py:5]: `create_base_params_airplane`

also, why do we want this to be in the examples? We already have example on how to create Flow360Params. We can use this as second example. But other examples should not refer to this file. If they need to load example config, it needs to be loaded from file. Every example should be standalone runnable script

File: examples/simulation_examples/cylinder_case.py
Comment (maciej-flexcompute) [inline examples/simulation_examples/cylinder_case.py:1]: same as ariplane_case.py comment

File: flow360/component/surface_mesh.py [hunk 2]
```diff
@@ -35,13 +38,46 @@ class SurfaceMeshDownloadable(Enum):
     CONFIG_JSON = "config.json"
 
 
+class SurfaceMeshFileFormat(Enum):
+    """
+    Surface mesh file format
+    """
+
+    UGRID = "lb8.ugrid"
+    STL = "stl"
```
Comment (maciej-flexcompute) [inline flow360/component/surface_mesh.py:47]: a test is needed for this class

File: examples/simulation_examples/airplane_case.py (hunk 1 above)
Comment (feilin-flexcompute) [reply examples/simulation_examples/airplane_case.py]: fixed.

File: examples/simulation_examples/cylinder_case.py
Comment (feilin-flexcompute) [reply examples/simulation_examples/cylinder_case.py:1]: fixed

**Review by maciej-flexcompute — APPROVED** [review]

**Pull request merged**